ThunderCloud/CircleProgressView.swift

```diff
@@ -28,13 +28,41 @@ open class CircleProgressView: UIView {
         }
     }
     
-    /// Apply default shadow configuration
+    /// Apply shadow configuration based on `shadowColor`, `shadowOpacity`, `shadowOffset`, `shadowRadius` properties
     public var hasShadow = true {
         didSet {
             updateShadow()
         }
     }
     
+    /// The shadow colour, defaults to .lightGray
+    var shadowColor: UIColor = .lightGray {
```

Comment:
Don't we have `shadowComponents` in `ThunderBasics` now ?

Reply:
Yep, but I couldn't think of any nice way of making that work with the concept of `hasShadow` I don't really want to remove the `hasShadow` property as it's public and therefore that would be a breaking change... Because of the way `shadow: ShadowComponents` works (It's an extension) you can't have a `didSet` and therefore you can't store the current `ShadowComponents` for re-applying if the user toggles `hasShadow`... Any ideas?

Reply:
I'm being stupid! I'm with you now :D